pkgs/by-name/uf/ufbt/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchPypi,
+  makeWrapper,
+  setuptools,
+  setuptools-git-versioning,
+  flipperzero-toolchain,
+}:
+
+buildPythonPackage rec {
+  pname = "ufbt";
+  version = "0.2.5";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-pQI8pSn5X6ISJ2rlEIfe6je4g7PY8HhKITl1GemMvf8=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = [
+    setuptools
+    setuptools-git-versioning
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/ufbt \
+      --set FBT_TOOLCHAIN_PATH ${flipperzero-toolchain}/opt/flipperzero-toolchain
+  '';
+
+  meta = with lib; {
+    description = "uFBT is a cross-platform tool for building applications for Flipper Zero. It is a simplified version of Flipper Build Tool (FBT).";
+    mainProgram = "ufbt";
+    homepage = "https://github.com/flipperdevices/flipperzero-ufbt";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ CodeRadu ];
+    platforms = platforms.linux;
+  };
```

```suggestion
  meta = {
    description = "uFBT is a cross-platform tool for building applications for Flipper Zero. It is a simplified version of Flipper Build Tool (FBT).";
    mainProgram = "ufbt";
    homepage = "https://github.com/flipperdevices/flipperzero-ufbt";
    license = licenses.gpl3Only;
    maintainers = with lib.maintainers; [ CodeRadu ];
    platforms = lib.platforms.linux;
  };
```
Also please change the description to fit https://nixos.org/manual/nixpkgs/stable/#var-meta-description